.github/workflows/test.yml

```diff
@@ -3,57 +3,122 @@ name: Run DTS tests
 on:
   pull_request:
     branches:
-      - 'main'
+      - "main"
+
+concurrency:
+  group: ${{ github.workflow }}
+
 jobs:
-  build-dts:
-    uses: ./.github/workflows/build.yml
-    with:
-      cacheless: false
+  # build-dts:
+  #   uses: ./.github/workflows/build.yml
+  #   with:
+  #     cacheless: false
+
+  checkout-osfv:
+    runs-on: dts-builder
+    steps:
+      - name: Checkout OSFV repo
+        uses: actions/checkout@v4
+        with:
+          repository: "Dasharo/open-source-firmware-validation"
+          path: "open-source-firmware-validation"
+          submodules: "recursive"
+          ref: "develop"
+
   run-tests:
     name: Run DTS tests
-    needs: build-dts
+    needs: [checkout-osfv]
     runs-on:
       labels: dts-builder
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - test: msi_comm
+            dpp_download_key: $MSI_DOWNLOAD
+            dpp_logs_key: $MSI_LOGS
+            log_file: output_msi.log
+
+          - test: msi_dpp
+            dpp_download_key: $MSI_DOWNLOAD
+            dpp_logs_key: $MSI_LOGS
+            log_file: output_msi.log
+
+          - test: msi_heads
+            dpp_download_key: $MSI_HEADS_DOWNLOAD
+            dpp_logs_key: $MSI_HEADS_LOGS
+            log_file: output_msi_heads.log
+
+          - test: optiplex_dpp
+            dpp_download_key: $OPTIPLEX_DOWNLOAD
+            dpp_logs_key: $OPTIPLEX_LOGS
+            log_file: output_nc_heads.log
+
+          - test: novacustom_heads
+            dpp_download_key: $OPTIPLEX_DOWNLOAD
+            dpp_logs_key: $OPTIPLEX_LOGS
+            log_file: output_nc_heads.log
+
+          - test: novacustom_comm
+            dpp_download_key: $OPTIPLEX_DOWNLOAD
+            dpp_logs_key: $OPTIPLEX_LOGS
+            log_file: output_nc_heads.log
+
+          - test: pcengines_dpp
+            dpp_download_key: $PCENGINES_DOWNLOAD
+            dpp_logs_key: $PCENGINES_LOGS
+            log_file: output_pcengines.log
+
+          - test: pcengines_seabios
```

Comment:
There is no separate `pcengines_seabios` tag

```
Suite 'Dts-E2E' contains no tests matching tag 'pcengines seabios'.
```

Reply:
@m-iwanicki this fixed in https://github.com/Dasharo/open-source-firmware-validation/pull/636

Reply:
I also fixed some mismathed arguments in the matrix configuration